### PR TITLE
Update Axelar testnet network and channels

### PR DESCRIPTION
**Pull request by daniel-farina**
As documented here: https://docs.axelar.dev/resources/testnet#ibc-channels

The network id for osmosis is `osmosis-5` and there are new channels  for the testnet. This is part of fixing the network configuration for the testnet. 

@jonator  Please let me know if there are any other places we might need to update. 

Please don't merge yet, I will do some test with the preview link first.

### CODE REVIEW

**Comment by JeremyParish69** [comment]
@daniel-farina some testnet transfers have worked in the past. Have they stopped working? Usually these shouldn't need changing. Governance can usually restore them. In the case the new channel-ids are to be used, 19XX is a really high number--perhaps confused with connection number?

File: packages/web/config/ibc-assets.ts [hunk 1]
```diff
@@ -27,8 +27,8 @@ export const IBCAssetInfos: (IBCAsset & {
     counterpartyChainId: IS_TESTNET
       ? "axelar-testnet-lisbon-3"
       : "axelar-dojo-1",
-    sourceChannelId: IS_TESTNET ? "channel-312" : "channel-208",
-    destChannelId: IS_TESTNET ? "channel-22" : "channel-3",
+    sourceChannelId: IS_TESTNET ? "channel-135" : "channel-1946",
```
Comment (JeremyParish69) [inline packages/web/config/ibc-assets.ts:30]: leave the prod channel id as channel-208

File: packages/web/config/ibc-assets.ts [hunk 2]
```diff
@@ -27,8 +27,8 @@ export const IBCAssetInfos: (IBCAsset & {
...
+    destChannelId: IS_TESTNET ? "channel-1946" : "channel-3",
```
Comment (JeremyParish69) [inline packages/web/config/ibc-assets.ts:31]: the testnet channel ids should be flipped for all 4 of these assets--source (osmosis side) as 1946 and dest (axelar side) as 135

**Comment by jonator** [comment]
@JeremyParish69 should be able to guide you on the config changes

File: packages/web/config/ibc-assets.ts (hunk 1 above)
Comment (daniel-farina) [reply packages/web/config/ibc-assets.ts:30]: > leave the prod channel id as channel-208
My bad. Good catch! ty

File: packages/web/config/ibc-assets.ts (hunk 2 above)
Comment (JeremyParish69) [reply packages/web/config/ibc-assets.ts]: should be corrected now

Comment (daniel-farina) [reply packages/web/config/ibc-assets.ts]: ok, I'll test it!

Comment (daniel-farina) [reply packages/web/config/ibc-assets.ts]: @JeremyParish69  can we also add Axelar on the testnet? We should probably have a testnet for frontier as well @jonator .

Comment (jonator) [reply packages/web/config/ibc-assets.ts]: If anything, the testnet should be frontier, since it includes all assets from main. This would be simpler

**Review by JeremyParish69 — COMMENTED** [review]
With the addition of gasPriceStep for Testnet, we should finally be good for this to work. But I believe @daniel-farina is doing transfer validation, is that right?

**Review by JeremyParish69 — APPROVED** [review]

**Pull request merged**